ui/src/server/handlers/api.js

```diff
@@ -1510,15 +1510,16 @@ Fetchr.registerService({
                 break;
             }
             case 'policy': {
-                req.clients.zms.putPolicyVersion(
-                    {
-                        domainName: params.domainName,
-                        policyName: params.collectionName,
-                        policy: params.detail,
-                        auditRef: params.auditRef,
-                    },
-                    responseHandler.bind({ caller: 'putPolicy', callback, req })
-                );
+                    req.clients.zms.putPolicy(
+                        {
+                            domainName: params.domainName,
+                            policyName: params.collectionName,
+                            policy: params.detail,
+                            auditRef: params.auditRef,
+                            returnObj: true,
+                        },
+                        responseHandler.bind({ caller: 'putPolicy', callback, req })
+                    );
```

Comment:
Looks good, I'm assuming tested works and the service tags fix will come in a separate pr?

Reply:
addressed comments and built on top of this pr in https://github.com/AthenZ/athenz/pull/2358